lib/tasks/evm.rake

```diff
@@ -3,6 +3,22 @@ require 'evm_application'
 require 'evm_rake_helper'
 
 namespace :evm do
+  namespace :foreman do
+    task :start => [:environment, 'db:seed'] do
+      server = MiqServer.my_server
+
+      # Assign and activate the default roles
+      server.ensure_default_roles
+      server.activate_roles(server.server_role_names)
+
+      # Mark the server as started
+      server.update_attributes(:status => "started")
```

Comment:
I think the idea is that this will become `evm:start` when we hit the "H" release.

The idea for this PR was to introduce the way that developers might run our processes when all the server knows how to do is talk to the kubeclient to start containers. So I'm not as concerned about if this plays nicely with the existing ways to start/stop the server.